dvc/remote/hdfs.py

```diff
@@ -20,6 +21,8 @@ class RemoteHDFS(RemoteBase):
     REGEX = r"^hdfs://((?P<user>.*)@)?.*$"
     PARAM_CHECKSUM = "checksum"
```

Comment:
So if you have SUPPORTED_CHECKSUM_TYPES down below, what do you need PARAM_CHECKSUM ?